src/posts/2022-02-10-design-het-web.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Design het Web'
+category: inspiration
+tags: new blogpost
+comments: true
+author: joris
+min_read: 5
+date: 2022-02-10
+permalink: /2022/02/10/design-het-web/
+---
+
+<!-- Section: Intro -->
+
+Development en webdesign zijn termen die vaak door elkaar worden gehaald. Hoewel het twee aparte taken zijn varen ze wel in hetzelfde vaarwater en zullen ze in een groot deel van de web-trajecten overlappen. Uit mijn ervaring blijkt dat veel personen die in één van deze twee categorieën werken ook behendig zijn in de ander, maar hebben altijd voorkeur voor hun eigen onderdeel. Welke mogelijkheden zijn er voor wie als we het hebben over doorgroeien(als designer)?
+
+Er zijn een aantal mogelijkheden waar je uit kan kiezen als designer: je wil doorleren en de code echt gaan beheersen, je wilt omscholen en het design achter je laten, je vindt allebei leuk en blijft er tussenin hangen of je sluit de code helemaal uit.
+
+## Vergeet het design
+
+Er is een mogelijkheid om het designen helemaal te laten voor wat het is en je vol te storten op de code. Dit is iets wat ik niet zou aanraden; veel bedrijven zijn blij met developers die ook designkennis hebben. Om binnen organisaties de overlap tussen design en development zo soepel mogelijk te laten lopen zijn een aantal, of tenminste één zo’n persoon nodig. Je kunt dus een belangrijke rol gaan spelen.
+
+In het begin heb je nog niet veel aan je design skills, dit komt omdat je nog heel beperkt bent in wat je kunt maken. In een later stadium zul je gaan merken dat je niet alleen vette dingen kunt maken, maar ook verzinnen. Je kunt dan nauw samenwerken met de designers, en ze zullen ook blij zijn met jou.
+
+## Leer het bij: Code, Code, Code!
+
+In het begin is het niet te overzien; verschillende tools, talen & frameworks; Vue, Angular, React etc!
+Ook worden er veel fancy Engelse namen en code talen gebruikt in dagelijks communicatie. Afhankelijk van je doel is de learning curve best stijl, zeker als je alleen een design achtergrond hebt. Er zijn ook veel overeenkomsten, die zie je vooral terug in HTML/CSS. Denk aan eigenschappen van elementen, werkstructuur en het uiteindelijke doel; gave dingen maken!
+
+Ik heb gemerkt dat het iets is wat je over je heen moet laten komen. Veel stof is logischer en minder ingewikkeld dan hoe het er in eerste instantie uitziet. Als designer heb je ook te maken met projectmatig werken, het einddoel waarborgen en een professioneel product afleveren. De tooling is alleen anders.
+Het moeilijkste om te leren is het denken als een developer; een soort analytische, oplossingsgerichte visie die je moet hebben, naast je algemene programmeer vaardigheden natuurlijk. Met genoeg oefening is dit zeker onder de knie te krijgen en moeten de meeste designers met een beetje geduld, hier wel uit gaan komen.
+
+<figure>
+    <img src="/assets/design-het-web/code.jpg" alt="Code">
+    <figcaption>Leer project HTML/CSS</figcaption>
+</figure>
```

Comment:
Rather difficult screenshot to read (small things in there).
Perhaps only show what is relevant.
Or if the entire editor with files & code are; size things up a bit 🙂